charts/flagger/values.yaml

```diff
@@ -102,3 +102,6 @@ tolerations: []
 prometheus:
   # to be used with ingress controllers
   install: false
+
+kubeconfigSource: ""
+kubeconfigHost: "kubeconfig"
```

Comment:
This will break everyones install, please set it to string empty, also please add a comment and explain it does.